test/classes/claridge/baseConstructorCall.chpl

```diff
@@ -1,17 +1,18 @@
 class BaseClass {
   var a, b: int;
 
-  proc BaseClass(b: int) {
+  proc init(b: int) {
     this.b = b;
+    super.init();
```

Comment:
I think if the test requires a super.init() then putting it in the most natural dividing line between "initialization" and "general computation" makes sense (and I realize that the two tests of mine in this PR _could_ be considered of that flavor, though I could also imagine a user naively thinking of the array assignment as being initialization-like even though it doesn't meet our technical definition).

I don't think we should force tests to use phase 1 for the purposes of finding bugs in the implementation if they don't "want" it naturally.  I.e., our main goal should be to convert the tests over so that they keep working in a deprecated initializer world and to be as true to the test as possible.  Personally, if I had a test in the system that looked like this:

```chapel
class C {
  var x: int;
  proc C() {
    x = 42;
  }
}
```
and I were tasked with rewriting it, I'd rewrite it as:

```chapel
proc init() {
  x = 42;
}
```

(but I'm also a person who didn't understand why C++ supported initializer lists for years and avoided using them as a result).  Even understanding them, my preference is not to use super.init when it's unnecessary and hope that the compiler would optimize the duplicate initialization away (the back-end compiler, if not ours).

This might be a reason to focus initially not just on the most complicated tests, but also the ones that don't have an obvious owner.  That way, the owner can make the call as to how to rewrite them (assuming they have the cycles/interest in doing the rewrite).